addon/components/base/bs-form/element/control/input.js

```diff
@@ -32,7 +32,8 @@ export default Control.extend({
     'step',
     'form',
     'spellcheck',
-    'title'
+    'title',
+    'ariaDescribedby:aria-describedBy'
```

Comment:
That's assuming we have control of when and what kind of controls are added. But I actually worked the past week on a future `ember-bootstrap-power-select` addon, that just adds some more convenient support for the `power-select` component to our forms. It works by adding a `bs-form/element/control/power-select` component that glues them together. This works because of #299, where it automatically picks up this new component when you use `{{form.element controlType="power-select"}}`. Have not pushed this yet, but I can do so later.

So this is basically an external component, that `ember-bootstrap`  is not really aware of. And when we make this way of extending form control "public", we cannot assume what controls we have...